watertap/unit_models/gac.py

```diff
@@ -1134,6 +1135,9 @@ def initialize_build(
         # --------------------------------------------------------------------
         # solve unit
 
+        # pre-solve using interval arithmetic
+        interval_initializer(self)
```

Comment:
@bknueven Does this warrant deprecating any initialization in the unit model that was previously necessary but now unused?

Reply:
I am a little confused by the question. Generally speaking, this method finds a better initial point for Ipopt, and only in very simple cases can it actually solve the system of equations.

Reply:
So in the unit model, there are (sometimes custom) initialization routines that would try to better define the initial point. Are these directly overwriting those?

Not a great example, but this code [watertap/unit_models/gac.py#L1120](https://github.com/watertap-org/watertap/blob/617fe3a48bfbf081186d183a5ff619f4142a3a2f/watertap/unit_models/gac.py#L1120) overwrites the default initial values for the molar flow of inert species to 0 from whatever the default value they are set to from the property model block because I know from the unit model constraints they are going to evaluate those variables to 0. 

I'm assuming this is what `interval_initializer()` is doing but in a more automated wider scope approach. After looking at the code, maybe it looks like it is more just adjusting the initial guess such that all initial values begin within their defined bounds as opposed to actually providing a smart initial value?

Reply:
I think it could replace some of these custom initialization routines. It will overwrite them only if it can prove that the value lies outside the initial value provided.

The code is simple because everything non-trivial is in the call to `fbbt`. You can read about the underlying algorithm [here in short form](https://octeract.gg/docs/blog/domain-reduction/feasibility-based-bounds-tightening-fbbt/) and [here in long form](https://optimization-online.org/wp-content/uploads/2012/01/3325.pdf).